### PR TITLE
[Update] Update note for Getting Started with PyTorch on Intel GPUs

### PR DESCRIPTION
**Pull request by ZhaoqiongZ**
remove the hardware and software prerequisites and set up env part.
keep the prerequisites section and link to pytorch prerequistes for intel gpus for driver install, intel support package install and env set up
https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpus.html
Update the support for Intel Client GPU MTL-H
Update inference & training examples

### CODE REVIEW

**Comment by ZhaoqiongZ** [comment]
Hi @malfet , I updated the document get_start_xpu for release PyTorch 2.5.
We add support for Client GPU MTL-H and I also updated the inference &  training script in examples.

**Comment by ZhaoqiongZ** [comment]
Hi @svekars , could you help review and merge the updates for the blog Getting Started with PyTorch on Intel GPUs

**Comment by ZhaoqiongZ** [comment]
Hi @svekars  @malfet , could any of you help on review this pr?

File: docs/source/notes/get_start_xpu.rst [hunk 1]
```diff
@@ -1,109 +1,81 @@
...
+""""""""""""""
 
-Before you begin, you need to set up the environment. This can be done by sourcing the ``setvars.sh`` script provided by the ``intel-for-pytorch-gpu-dev`` and  ``intel-pti-dev`` packages.
+
+Now we have all the required packages installed and environment activated. Use the following commands to install ``pytorch``, ``torchvision``, ``torchaudio`` on Linux.
 
 .. code-block::
 
-   source ${ONEAPI_ROOT}/setvars.sh
+    pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/xpu
```
Comment (malfet) [inline docs/source/notes/get_start_xpu.rst:43]: I get HTTP/403 when trying to access https://download.pytorch.org/whl/xpu

Comment (ZhaoqiongZ) [reply docs/source/notes/get_start_xpu.rst:43]: Hi @malfet , for now we use pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/xpu, the link in the blog is official one which will be available later.

File: docs/source/notes/get_start_xpu.rst [hunk 2]
```diff
@@ -1,109 +1,81 @@
...
-     - Intel® Data Center GPU Max Series
-   * - Supported OS
+   * - Validated Hardware
+     - Supported OS
+   * - Intel® Data Center GPU Max Series
      - Linux
+   * - Intel Client GPU
+     - Windows/Linux
+
+Intel GPUs support enhancement is ready in PyTorch* 2.5 to provide Beta quality for Intel® Data Center GPU Max Series and Intel® Client GPUs on both Linux and Windows, which brings Intel GPUs and the SYCL* software stack into the official PyTorch stack with consistent user experience to embrace more AI application scenarios, especially in AIPC scene.
```
[the remark this answers is not in the document]
Comment (ZhaoqiongZ) [reply docs/source/notes/get_start_xpu.rst:18]: Hi @rivergold , please help clarify here.

[the remark this answers is not in the document]
Comment (ZhaoqiongZ) [reply docs/source/notes/get_start_xpu.rst]: Hi @malfet , I remove the confusing word beta quality and AIPC in the blog.

**Review by malfet — COMMENTED** [review]
Please modify the PR to contain correct link and then it's good to go.

**Comment by ZhaoqiongZ** [comment]
Hi @malfet , Now I've change the installation link to the correct one!

File: docs/source/notes/get_start_xpu.rst [hunk 3]
```diff
@@ -1,109 +1,81 @@
...
-     - Intel® Data Center GPU Max Series
-   * - Supported OS
+   * - Validated Hardware
+     - Supported OS
+   * - Intel® Data Center GPU Max Series
      - Linux
+   * - Intel Client GPU
+     - Windows/Linux
+
+Intel GPUs support enhancement is ready in PyTorch* 2.5 for Intel® Data Center GPU Max Series and Intel® Client GPUs on both Linux and Windows, which brings Intel GPUs and the SYCL* software stack into the official PyTorch stack with consistent user experience to embrace more AI application scenarios.
```
Comment (seemethere) [inline docs/source/notes/get_start_xpu.rst:18]: Would it be good to call out that this support is still in an experimental / beta phase considering 2.5 is the first release with actual binaries?

Also considering we don't have a consistent CI with these builds I'd be worried that we're giving users the wrong impression that these are very well battle tested

**Review by seemethere — CHANGES_REQUESTED** [review]

File: docs/source/notes/get_start_xpu.rst (hunk 3 above)
Comment (ZhaoqiongZ) [reply docs/source/notes/get_start_xpu.rst:18]: Sure, let me add it.

Comment (ZhaoqiongZ) [reply docs/source/notes/get_start_xpu.rst]: Done

Comment (ZhaoqiongZ) [reply docs/source/notes/get_start_xpu.rst]: Hi @seemethere , here is the list of CI for Intel GPU 
https://github.com/pytorch/pytorch/blob/main/.github/workflows/xpu.yml

**Review by seemethere — APPROVED** [review]

**Comment by ZhaoqiongZ** [comment]
Hi @malfet , all the comments have been addressed, could you help approve this one?

File: docs/source/notes/get_start_xpu.rst (hunk 1 above)
Comment (malfet) [reply docs/source/notes/get_start_xpu.rst:45]: I'm not sure this is the case, but sure,  if you insist on pointing the users to broken link, this is your choice.

File: docs/source/notes/get_start_xpu.rst [hunk 4]
```diff
@@ -1,109 +1,96 @@
...
-Now we have all the required packages installed and environment acitvated. Use the following commands to install ``pytorch``, ``torchvision``, ``torchaudio`` by building from source. For more details, refer to official guides in `PyTorch from source <https://github.com/pytorch/pytorch?tab=readme-ov-file#intel-gpu-support>`_, `Vision from source <https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#development-installation>`_ and `Audio from source <https://pytorch.org/audio/main/build.linux.html>`_.
+Now we have all the required packages installed and environment activated. Use the following commands to install ``pytorch`` on Windows, build from source for ``torchvision`` and ``torchaudio``.
+
+For release wheels
+
+.. code-block::
+
+    pip3 install torch --index-url https://download.pytorch.org/whl/xpu
+
+For nightly wheels
```
Comment (malfet) [inline docs/source/notes/get_start_xpu.rst:64]: If installation instructions are the same for Linux and Windows, not sure what are the benefit of having two separate sections

**Review by malfet — COMMENTED** [review]
Thank you for making the change. IMO there are still some areas of improvement, but sounds fine to me as is right now

**Pull request closed without merging**